src/tool/hpcrun/gpu/gpu-metrics.c

```diff
@@ -583,6 +606,16 @@ gpu_metrics_default_enable
 
   FINALIZE_METRIC_KIND();
 
+// Execution count metrics
+#undef CURRENT_METRIC 
+#define CURRENT_METRIC GCOUNT
+
+  INITIALIZE_METRIC_KIND();
+
+  FORALL_GCOUNT(INITIALIZE_SCALAR_METRIC_INT)
+
+  FINALIZE_METRIC_KIND();
+
 // Memcpy metrics
```

Comment:
The following metrics are for cudaMemoryAlloc/Free APIs but not memcpy APIs.

Reply:
And we could remove GMEM from the default metrics for now because actually we do not monitor memory alloc/dealloc events. CUPTI does not provide a correlationId field in CUpti_ActivityMemory.

Reply:
Not sure if roctracer API manifests correlation ids for memory alloc/free events.

Reply:
We can certainly remove GMEM from default metrics since we don't monitor them now.